src/ProtocolGateway.Core/Instrumentation/SafePerformanceCounter.cs

```diff
@@ -4,16 +4,28 @@
 namespace Microsoft.Azure.Devices.ProtocolGateway.Instrumentation
 {
     using System;
+#if !NETSTANDARD1_3
     using System.Diagnostics;
+#endif
 
-    public sealed class SafePerformanceCounter
+    public sealed class SafePerformanceCounter : IPerformanceCounter
```

Comment:
In netstandard is used just to wrap EmptyPerformanceCounter so that PerformanceCounters class has the same properties as before (for example SafePerformanceCounter ConnectionsEstablishedTotal = ManagerInstance.GetCounter(CategoryName, ConnectionsEstablishedTotalCounterName). This is to make sure it doesn't break public API.